addons/web/static/src/views/relational_model.js

```diff
@@ -325,12 +325,12 @@ export class Record extends DataPoint {
             this.resId = state.resId;
         }
         if (!this.resId) {
-            this.resId = this.model.nextVirtualId--;
+            this.resId = false;
         }
```

Comment:
give virtual records a virtualId?

Reply:
usage id = rec.resId ou rec.virtualId used for communication with server
recordId used for communication webclient side